ethereumj-core/src/main/java/org/ethereum/sharding/crypto/Sign.java

```diff
@@ -0,0 +1,48 @@
+package org.ethereum.sharding.crypto;
+
+import java.math.BigInteger;
+import java.util.Objects;
+
+/**
+ * Signature utilities
+ * Signature should be implemented using BLS
+ */
+public interface Sign {
+
+    /**
+     * Sign the message
+     */
+    Signature sign(byte[] msg, byte[] privateKey);
+
+    /**
+     * Verifies whether signature is made by signer with publicKey
+     */
+    boolean verify(Signature signature, byte[] publicKey);
+
+    /**
+     * Aggregates several signatures in one
+     */
+    Signature aggSigns(Signature[] signatures);
```

Comment:
I'd move it to a separate interface? what do you think? The product of `aggSigns` is not of `Signature` type, there should be another model for that.

Reply:
From the specs:
```
# BLS aggregate signature
    'aggregate_sig': ['int256']
```
and it looks like there are just 2 of such ints there, same way as for Signature